test-clients/src/main/java/com/hedera/services/bdd/suites/block/BlockSuite.java

```diff
@@ -0,0 +1,196 @@
+                                .nonce(1)
+                                .maxFeePerGas(50L)
+                                .gasLimit(1_000_000L)
+                                .via(secondCall)
+                                .hasKnownStatus(ResponseCodeEnum.SUCCESS)
+                ).then(
+                        withOpContext((spec, opLog) -> {
+                            final var firstBlockOp = getTxnRecord(firstCall);
+                            final var recordOp = getTxnRecord(secondCall);
+                            allRunFor(spec, firstBlockOp, recordOp);
+
+                            final var firstCallRecord = firstBlockOp.getResponseRecord();
+                            final var firstCallLogs = firstCallRecord.getContractCallResult().getLogInfoList();
+                            final var firstCallTimeLogData = firstCallLogs.get(0).getData().toByteArray();
+                            final var firstCallTimestamp = Longs.fromByteArray(
+                                    Arrays.copyOfRange(firstCallTimeLogData, 24, 32));
+
+                            final var secondCallRecord = recordOp.getResponseRecord();
+                            final var secondCallLogs = secondCallRecord.getContractCallResult().getLogInfoList();
+                            final var secondCallTimeLogData = secondCallLogs.get(0).getData().toByteArray();
+                            final var secondCallTimestamp = Longs.fromByteArray(
+                                    Arrays.copyOfRange(secondCallTimeLogData, 24, 32));
+
+                            assertEquals(firstCallTimestamp, secondCallTimestamp,
+                                    "Block timestamps should be equal");
+
+                        })
+                );
+    }
+
+    private HapiApiSpec BLCK_001_002_003_004_returnsCorrectBlockProperties() {
+        final var contract = "EmitBlockTimestamp";
+        final var firstBlock = "firstBlock";
+        final var secondBlock = "secondBlock";
+
+        return defaultHapiSpec("returnsCorrectBlockProperties")
+                .given(
+                        newKeyNamed(SECP_256K1_SOURCE_KEY).shape(SECP_256K1_SHAPE),
+                        cryptoCreate(RELAYER).balance(6 * ONE_MILLION_HBARS),
+                        cryptoTransfer(tinyBarsFromAccountToAlias(GENESIS, SECP_256K1_SOURCE_KEY, ONE_HUNDRED_HBARS))
+                                .via("autoAccount"),
+                        getTxnRecord("autoAccount").andAllChildRecords(),
+
+                        uploadInitCode(contract),
+                        contractCreate(contract)
+                ).when(
+                        ethereumCall(contract, "logNow")
+                                .type(EthTxData.EthTransactionType.EIP1559)
+                                .signingWith(SECP_256K1_SOURCE_KEY)
+                                .payingWith(RELAYER)
+                                .nonce(0)
+                                .maxFeePerGas(50L)
```

Comment:
This should be failing for insufficient fees.  IIRC gas price is ~71 in test environments.  We should fail this transaction unless there is an auto-allowance from the relayer.

Reply:
There is a [default allowance of five hbars](https://github.com/hashgraph/hedera-services/blob/2a89626b0823e3a31c40a6eb4cd7de030dd5a9a7/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/contract/HapiEthereumCall.java#L91) , which should cover the fees if gas price is not sufficient.